tests/testthat/test-renderer1-axis-angle-rotate.R

```diff
@@ -28,12 +28,19 @@ expect_rotate_anchor <- function(info, rotate, anchor){
   expect_match(rotated["style", ], paste("text-anchor:", anchor), fixed=TRUE)
   expect_match(rotated["transform", ], paste0("rotate(", rotate), fixed=TRUE)
   # http://stackoverflow.com/questions/442404/retrieve-the-position-x-y-of-an-html-element
-  tick_box <- remDr$executeScript('return document.getElementsByClassName("xaxis")[0].firstChild.getBoundingClientRect()')
-  title_box <- remDr$executeScript('return document.getElementsByClassName("xtitle")[0].getBoundingClientRect()')
-  expect_true(title_box$top >= tick_box$bottom)
+  
+  
+  tick_box_script <- 'document.getElementsByClassName("xaxis")[0].getBoundingClientRect().bottom;'
+  title_box_script <- 'document.getElementsByClassName("xtitle")[0].getBoundingClientRect().top;'
+    
+  # Evaluate scripts and get the values
+  tick_box <- remDr$Runtime$evaluate(tick_box_script, returnByValue = TRUE)$result$value
+  title_box <- remDr$Runtime$evaluate(title_box_script, returnByValue = TRUE)$result$value
+  #An offset of 2.5 is added because chromote aligns text with slight differences
```

Comment:
As we discussed during previous meeting, can you add more details to this magic number? what is the unit of it? px? 🤔

Reply:
> As we discussed during previous meeting, can you add more details to this magic number? what is the unit of it? px? 🤔

@Faye-yufan Yes. the unit is px. I have made a new commit of that file explaining more about offset value and also links to resources regarding that.